pages/sdk/onramp-kit/guides/stripe.mdx

```diff
@@ -1,119 +1,152 @@
-  ```
-
-  ### Initialize the StripePack
-
-  The `StripePack` can be used with any frontend framework like React, Vue, Angular or even plain HTML and JavaScript. In this example, we are using it with plain JavaScript.
-
-  Load the application and initialize the `StripePack` using the following snippet:
-
-  ```typescript
-  const stripePack = new StripePack({
-    // Get public key from Stripe: https://dashboard.stripe.com/register
-    stripePublicKey:
-      'pk_test_51MZbmZKSn9ArdBimSyl5i8DqfcnlhyhJHD8bF2wKrGkpvNWyPvBAYtE211oHda0X3Ea1n4e9J9nh2JkpC7Sxm5a200Ug9ijfoO',
-    // Deploy your own server: https://github.com/5afe/aa-stripe-service
-    onRampBackendUrl: 'https://aa-stripe.safe.global'
-  })
-  await stripePack.init()
-  ```
-
-  ### Render the Stripe widget
-
-  Add a container in your web app.
-
-  ```html
-  <div id="stripe-root"></div>
-  ```
-
-  We choose to add a `div` container with the `stripe-root` id to render the Stripe widget inside by providing the CSS selector to the `element` prop in the `open()` method.
-
-  ```typescript
-  // See options for using the StripePack open method in:
-  // https://stripe.com/docs/crypto/using-the-api
-
-  const sessionData = await stripePack.open({
-    element: '#stripe-root', // Can be any CSS selector
-    theme: 'light' // light | dark
-    // Optional, if you want to use a specific created session
-    // ---
-    // sessionId: 'cos_1Mei3cKSn9ArdBimJhkCt1XC',
-    // Optional, if you want to specify default options
-    // ---
-    // defaultOptions: {
-    // transaction_details: {
-    //   wallet_address: walletAddress,
-    //   lock_wallet_address: true
-    //   supported_destination_networks: ['ethereum', 'polygon'],
-    //   supported_destination_currencies: ['usdc'],
-    // },
-    // customer_information: {
-    //   email: 'john@doe.com'
-    // }
+### Obtain your public and private keys
+
+To use the Stripe Crypto Onramp service, you need to obtain your [public and private keys](https://docs.stripe.com/keys).
+You have to apply for the crypto onramp service and add at least your business address and information to your Stripe account.
+When your application is approved, you will find your public and private keys in your [Stripe Developer Dashboard](https://dashboard.stripe.com/test/apikeys).
+
+### Install the SDK and client library
+
+You can install the Stripe SDK as a script or as a package. 
+
+<CH.Section>
+  <CH.Code style={{boxShadow: 'none'}}>
+
+    ```html script
+    <head>
+      <title>Onramp</title>
+      <script src="https://js.stripe.com/v3/"></script>
+      <script src="https://crypto-js.stripe.com/crypto-onramp-outer.js"></script>
+    </head>
+    ```
+
+    ```bash package
+    npm install --save @stripe/stripe-js @stripe/crypto
+    ```
+  </CH.Code>
+</CH.Section>
+
+### Generate a crypto onramp session
+
+You need to generate a crypto onramp session using your stripe private key.
```

Comment:
Should these be numbered steps (if they should be done in order)?

Reply:
Hey, not sure which steps you mean here.

Reply:
The steps container creates numbered steps from the "###" headlines. Did you mean this?